test/Common/ServerFixture.cs

```diff
@@ -87,5 +70,24 @@ public void Dispose()
             _logger.LogInformation("Shutting down test server");
             host.Dispose();
         }
+
+        private class ForwardingLoggerProvider : ILoggerProvider
+        {
+            private readonly ILoggerFactory _loggerFactory;
+
+            public ForwardingLoggerProvider(ILoggerFactory loggerFactory)
+            {
+                _loggerFactory = loggerFactory;
+            }
+
+            public void Dispose()
+            {
```

Comment:
Actually the ForwardingLoggerProvider does not own the _loggerFactory. It should be disposed by the fixture who creates it.